src/main/java/de/hpi/bpt/argos/util/threading/BackgroundWorkerImpl.java

```diff
@@ -0,0 +1,54 @@
+package de.hpi.bpt.argos.util.threading;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.slf4j.LoggerFactory'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)